test/integration/ExpiringDataCleanup.test.ts

```diff
@@ -0,0 +1,35 @@
+import fetch from 'cross-fetch';
+import type { App } from '../../src/init/App';
+import { getPort } from '../util/Util';
+import {
+  getDefaultVariables,
+  getTestConfigPath,
+  instantiateFromConfig,
+} from './Config';
+
+jest.useFakeTimers('legacy');
+
+const port = getPort('ExpiringDataCleanup');
+const baseUrl = `http://localhost:${port}/`;
+
+describe('A server with expiring storage', (): void => {
+  let app: App;
+
+  beforeAll(async(): Promise<void> => {
+    const instances = await instantiateFromConfig(
+      'urn:solid-server:test:Instances',
+      getTestConfigPath('server-memory.json'),
+      getDefaultVariables(port, baseUrl),
+    ) as Record<string, any>;
+    ({ app } = instances);
+    await app.start();
+  });
+
+  it('does not crash.', async(): Promise<void> => {
+    // Default timeout is 1 hour
+    // This test would fail if something goes wrong in an interval timer
+    jest.advanceTimersByTime(2 * 60 * 60 * 1000);
+    const res = await fetch(baseUrl, { method: 'HEAD' });
```

Comment:
The problem is that there are no easy ways to check this that I can find, unless we manually create a full custom server setup with some mocks in there. The `fetch` is actually there to force promises to be resolved and since this server setup is fully in memory they should be able to execute immediately since there is no IO.

But it's definitely not perfect, this test is just a minor potential alarm in case something goes wrong again.

Reply:
Yeah, good enough.